File: tests/J2C_NoReturnResults/jt6.c
jasync localyou(int c, char *s) {
    while(1) {
        jsys.sleep(10000000);
        printf("############-->>> Hello YOU  %d, %s\n", c, s);
        testfunc(-10);
    }
}

jasync testfunc(int x) {
    printf("------------ Value of x %d\n", x);
}


int main(int argc, char *argv[]) {
    localyou(10, "local-message...");
    return 0;
}

File: tests/J2C_NoReturnResults/jt6.js

let count = 10;

async function sleep(n) {
    return new Promise((resolve)=> {
        setTimeout(()=> {
            resolve();
        }, n);
    });
}

while(1) {
    await sleep(1000);
    console.log("Hello... calling the C side... ");
    try {
        await testfunc(count++);
    } catch(e) {
        console.log("Error.. ", e.message);
    }
}
